app/models/admin/nav.rb

```diff
@@ -157,6 +157,12 @@ def misc
             path: referral_programs_admin_index_path,
             count: Referral::Program.count,
             count_type: :records
+          ),
+          make_item(
```

Comment:
At some point, we should add this to the Command Bar (cmd+k), but that's low priority at the moment.